ffi/src/winscard/scard_handle.rs

```diff
@@ -537,7 +537,7 @@ pub unsafe fn copy_io_request_to_scard_io_request(
 
     let pci_info_len = io_request.pci_info.len();
     // SAFETY: it's safe to deref because we've checked for null value above.
-    let scard_pci_info_len = unsafe { (*scard_io_request).cb_pci_length }.try_into()?;
+    let scard_pci_info_len = usize::try_from(unsafe { (*scard_io_request).cb_pci_length })?;
```

Comment:
style: I recommend splitting on two lines instead of having the unsafe block inlined.

Reply:
done